src/main/routes/receiver.ts

```diff
@@ -90,6 +91,12 @@ async function retrieveRedirectForLandingPage (user: User): Promise<string> {
   const claimIssuedButNoResponse: boolean = (claimAgainstDefendant).length > 0
     && !atLeastOneResponse
 
+  const draftResponse: Draft<ResponseDraft>[] = await new DraftService().find('response', '100', user.bearerToken, (value: any): ResponseDraft => {
```

Comment:
I don't think this is the fix, the claimIssuedButNoResponse case covers this.

This happens before there is a draft response

Reply:
Behaviour before change is as follows:

```
if (claimIssuedButNoResponse) {
    return ResponsePaths.taskListPage
      .evaluateUri({ externalId: claimAgainstDefendant.pop().externalId })
  }
```

There is redirect to task list in case response has not been submitted yet but another linking happened. Robert agreed solution with Mark that in that case redirect to dashboard should happen so that user can make decision what to do, whether continue response of the draft or start response for newly linked claim.

Reply:
right

Reply:
I think that solution has a chance to work but we agreed with Mark to take users always to dashboard to simplify it.